src/geo.c

```diff
@@ -608,6 +610,12 @@ void georadiusGeneric(client *c, int srcKeyIndex, int flags) {
                       flags & GEOSEARCH &&
                       !fromloc)
             {
+                if (zobj == NULL) {
+                    if ((flags & GEOSEARCHSTORE) && storekey) goto delete_store_key;
+
+                    goto return_emptyarray;
+                }
```

Comment:
don't you mean `||` rather than `&&`?
anyway, my advise is to have just one goto label, and add code there to test `flags` and `storekey`.
```suggestion
                if (zobj == NULL) goto return_emptyarray; /* Abort before trying to look for the member. */
```

anyway, considering the comment i gave about RADIUS_MEMBER, i think we better remove all the `goto`s from the argument parsing and have them all after the argument parsing.

if we do that, then i think we have only one `goto` left, so we can just put an early return there, and there's no need for `goto` labels at all.

Reply:
Yes. you are right, it's better. Thanks for the review.
I modified it according to your suggestions. Please check again
Test code is on the way.